### PR TITLE
Mileston 1

### PR DESCRIPTION
**Pull request by AbigiyaTY**
In this milestone I:

- Created an Author class and Game class 
- Created an Author class with an association to the Item class (in a separate .rb file).
-  Add the 'add_item' method in the Author
   -  takes an instance of the Item class as an input
   -  add the input item to the collection of items
   -  add self as a property of the item object (by using the correct setter from the item object)
- All Game class properties visible in the diagram are defined and set up in the constructor method.
- All Author class properties visible in the diagram are defined and set up in the constructor method.
- Created can_be_archived?() in the Game class
  -  override the method from the parent class
  -  return true if parent's method returns true AND if last_played_at is older than 2 years
  -  otherwise, it should return false
- Add unit tests for all implemented methods.

### CODE REVIEW

File: .rubocop.yml [hunk 1]
```diff
@@ -17,7 +17,7 @@ Metrics/ClassLength:
   Max: 150
 Metrics/BlockLength:
   IgnoredMethods: ['describe']
-  Max: 30
+  Max: 300
```
Comment (Farida-Hamid) [inline .rubocop.yml:20]: Thanks for fixing it for all of us.

File: item.rb [hunk 2]
```diff
@@ -35,4 +35,12 @@ def genre=(genre)
...
     @genre.items << self
   end
+
+  def create_json
+    {
+      id: @id,
+      published_date: @published_date,
+      archived: @archived
+    }
+  end
```
Comment (Farida-Hamid) [inline item.rb:45]: Again, thanks for helping all of us.

**Review by Farida-Hamid — APPROVED** [review]
### All requirements are met excellently.
You went beyond the requirements. That's inspiring.

Keep it up.

File: item.rb (hunk 2 above)
Comment (AbigiyaTY) [reply item.rb:45]: Thanks so much for your feedback.

**Pull request merged**